OpenRA.Game/Network/OrderIO.cs

```diff
@@ -31,13 +31,14 @@ public static List<Order> ToOrderList(this byte[] bytes, World world)
 			return ret;
 		}
 
-		public static byte[] SerializeSync(int sync)
+		public static byte[] SerializeSync(World.SyncHash sync)
```

Comment:
The `SyncHash` class is probably overcomplicating this, IMO. It would probably be better to change this method to `public static byte[] SerializeGameState(int sync, long players)`, and make similar changes on the unpacking side.

Reply:
Indeed, this makes the whole PR much simpler, done. Thanks.